install.sh

```diff
@@ -3,7 +3,11 @@
 # Exit if something fails
 set -e
 
-if [[ $(basename "$PWD") !=  "JetBrainsRunner"* ]];then
+if [ -d "$PWD" ]; then
```

Comment:
I meant checking "$PWD/JetBrainsRunner", if yes we go inside of it, update the project and proceed to build.

Not sure if I'd update the project when the install.sh script is run from the project dir, maybe leave that as it is for now.